content/installation/image-builder.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: bt_wiki
+title: Using the Image Builder
+category: Installation
+draft: false
+weight: 600
+
+---
+
+{{% gsNote title="Supported Clouds" %}}
+ * AWS
+ * OpenStack
+ * vSphere
+{{% /gsNote %}}
+
+{{% gsNote title="Prerequisites" %}}
+ * [packer](https://www.packer.io/intro/getting-started/setup.html) += 0.8.6 (older versions not tested)
+ * Account credentials for each cloud platform you want to build on.
+{{% /gsNote %}}
+
+
+ 1. Clone the image bakery repo
+ 2. cd into the image bakery path
+    ```bash
+    git clone https://github.com/cloudify-cosmo/cloudify-image-bakery.git
+    cd cloudify-image-bakery/cloudify_marketplace
+    ```
+
+ 3. Copy inputs-example.json and edit with your favourite editor
+    ```bash
+    cp inputs-example.json inputs.json
+    $EDITOR inputs.json
```

Comment:
Yeah, turns out *I* have $EDITOR defined because I added it myself. I initially wanted to use `editor` but OSX doesn't have that.